packages/components/src/FilterDateRangePicker/subcomponents/DateRangeInputField/DateRangeInputField.module.scss

```diff
@@ -9,7 +9,7 @@ $input-disabled-opacity: 0.3;
   border: none;
 }
 
-.inputStartDate {
+.inputStartDate input {
```

Comment:
Moved `classNameOverride` in DateInput a level up to the wrapper to allow for stickersheet pseudo state tests